lib/collectionspace/converter/core/acquisition.rb

```diff
@@ -9,73 +9,100 @@ def convert
           end
         end
 
-        def self.map(xml, attributes)
-          CSXML.add xml, 'acquisitionReferenceNumber', attributes["acquisitionreferencenumber"]
-
-          CSXML.add_group xml, 'accessionDate', {
-            "dateDisplayDate" => CSDTP.parse(attributes["accessiondategroup"]).display_date,
-            "dateEarliestScalarValue" => CSDTP.parse(attributes["accessiondategroup"]).earliest_scalar,
-            "dateLatestScalarValue" => CSDTP.parse(attributes["accessiondategroup"]).latest_scalar,
+       def self.map(xml, attributes)
+         pairs = {
+            'acquisitionreferencenumber' => 'acquisitionReferenceNumber',
+            'acquisitionauthorizer' => 'acquisitionAuthorizer',
+            'acquisitionauthorizerdate' => 'acquisitionAuthorizerDate',
+            'acquisitionmethod' => 'acquisitionMethod',
+            'acquisitionnote' => 'acquisitionNote',
+            'acquisitionprovisos' => 'acquisitionProvisos',
+            'acquisitionreason' => 'acquisitionReason',
+            'creditline' => 'creditLine',
+            'grouppurchasepricecurrency' => 'groupPurchasePriceCurrency',
+            'grouppurchasepricevalue' => 'groupPurchasePriceValue',
+            'objectofferpricecurrency' => 'objectOfferPriceCurrency',
+            'objectofferpricevalue' => 'objectOfferPriceValue',
+            'objectpurchaseofferpricecurrency' => 'objectPurchaseOfferPriceCurrency',
+            'objectpurchaseofferpricevalue' => 'objectPurchaseOfferPriceValue',
+            'objectpurchasepricecurrency' => 'objectPurchasePriceCurrency',
+            'objectpurchasepricevalue' => 'objectPurchasePriceValue',
+            'originalobjectpurchasepricecurrency' => 'originalObjectPurchasePriceCurrency',
+            'originalobjectpurchasepricevalue' => 'originalObjectPurchasePriceValue',
+            'transferoftitlenumber' => 'transferOfTitleNumber'
           }
+          pairstransforms = {
```

Comment:
@Paulmulonzia The transforms hash should be a separate variable like this, NOT embedded as a hash in the CSXML::Helpers or CSXML function arguments.

Reply:
Got it, thanks.